ios/RNFirebase/links/RNFirebaseLinks.m

```diff
@@ -2,18 +2,99 @@
 #import "Firebase.h"
```

Comment:
I didn't find anywhere is this file where the link is resolved

Reply:
Indeed, the link is not resolved here. Only the initial. The link is treated in the appdelegate.m file

Reply:
Shouldn't we treat the link the library (like we do in Android?)

Reply:
how exactly? There is no equivalent to getDynamicLink.
the event is sent to JS via sendDynamicLink.

Reply:
Can't we move this code to the library?
```
[[FIRDynamicLinks dynamicLinks]
   handleUniversalLink:userActivity.webpageURL
   completion:^(FIRDynamicLink * _Nullable dynamicLink, NSError * _Nullable error) {
     if (dynamicLink) {
       [RCTLinkingManager application:application
                              openURL:dynamicLink.url
                    sourceApplication:nil
                           annotation:@{}];
       userActivity.webpageURL = dynamicLink.url;
     }
   }];
```